### PR TITLE
New Juju mocks

### PR DESCRIPTION
**Pull request by mullikine**
```
| name                             | hasIssues |
|----------------------------------+-----------|
| constructor-returns-interface    | false     |
| interface-next-to-implementation | true      |
| interface-too-many-methods       | true      |
| use-pointer-arg-for-worker       | false     |
```

### CODE REVIEW

File: tenets/codelingo/juju/api-not-closed/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#api-not-closed
+
+_by codelingo, one of their juju bundles_
+
+
+[Load up in the codelingo.io/playground](https://codelingo.io/playground/?repo=github.com/codelingo/hub&dir=tenets/codelingo/juju/api-not-closed&tenet=codelingo/juju/api-not-closed)
```
Comment (waigani) [inline tenets/codelingo/juju/api-not-closed/README.md]: Remove all this links.

**Comment by mullikine** [comment]
I have removed the real tenets from this PR.

**Comment by mullikine** [comment]
The 4 real juju tenets are branched from this commit.

File: registry/tenets.yaml [hunk 2]
```diff
@@ -155,8 +155,148 @@ codelingo:
...
       - golang
       - go
+  juju:
+    description: Best practices for the juju/juju repository
+    version: 0.0.0
+    tenets:
+      global-var:
+        hasIssues: false
+      exported-var-outside-export-test:
+        hasIssues: false
```
Comment (waigani) [inline registry/tenets.yaml]: All mocks should have an issue, which is that the CLQL query is not implemented.

File: tenets/codelingo/juju/api-not-closed/codelingo.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+funcs:
+  - name: appendIntegrationTest
```
Comment (waigani) [inline tenets/codelingo/juju/api-not-closed/codelingo.yaml]: remove this func from the mocks

**Pull request closed without merging**